src/main/clm_varctl.F90

```diff
@@ -227,6 +227,8 @@ module clm_varctl
   !
   real(r8), public :: nfix_timeconst = -1.2345_r8 
 
+  character(len=25), public :: nfix_method  ! choice of nfix parameterization
```

Comment:
Since, this is only needed as part of CNFUNMod.F90, it should be a local module variable there, with a ReadNML method that's in there as well. This reduces unneeded dependencies,, makes modules more independent, more interoperable, and easier to extract or add in.

Reply:
Ok, done.